lib/config.ts

```diff
@@ -3,16 +3,30 @@ import path from "path";
 import url from "url";
 import yaml from "js-yaml";
 
+import output from "./output";
 import consts from "./consts";
 import { Project, ConfigYAML } from "./types";
 
-function createFileIfMissing(filename: string) {
+export const DEFAULT_CONFIG_JSON: ConfigYAML = {
+  sources: {
+    components: true,
+  },
+  variants: true,
+  format: "flat",
+};
+
+export const DEFAULT_CONFIG = yaml.dump(DEFAULT_CONFIG_JSON);
+
+function createFileIfMissing(filename: string, defaultContents?: any) {
```

Comment:
Curiosity question - I wonder if it's possible/worth it to do input validation on the config file via zod

Reply:
Something I'd love to do in the near future!